cluster/src/dunit/scala/io/snappydata/externalstore/ColumnTableDUnitTest.scala

```diff
@@ -550,8 +550,8 @@ class ColumnTableDUnitTest(s: String) extends ClusterManagerTestBase(s) {
 
     assert(r.length == 10012)
 
-    println("startSparkJob6 " + region.size())
-    println("startSparkJob6 " + shadowRegion.size())
+    println("startSparkJob7 " + region.size())
```

Comment:
Not related to this PR, but this is inside def startSparkJob7 and not startSparkJob6.